src/libs/api.ts

```diff
@@ -59,7 +59,7 @@ function respondSuccess(res: Response, data: any = {}, action: Action | null = n
 function respondError(res: Response, error: Error | string | null, action: Action | null = null) {
 	// Check for different types of errors
 	let err = null;
-	if (error !== null && typeof error === 'object' && typeof error.message === 'string') {
+	if (error !== null && typeof error === 'object') {
```

Comment:
Not in this case, since `typescript-is` doesn't support checking for classes, so it would either have to be `error instanceof Error` or `typeof error === 'object'`.